pysc2/lib/renderer_human.py

```diff
@@ -676,6 +677,25 @@ def draw_overlay(self, surf):
     surf.surf.blit(text, (3, 3))
 
   @sw.decorate
+  def draw_shield_energy_status(self, surf):
+    """Draw the energy and shield status under the unit."""
+    for u, p in self._visible_units():
+      if self._camera.intersects_circle(p, u.radius):
+        if u.alliance != 3:
+          if u.energy or u.shield:
+            text = self.font_small.render(
+                "En: %.0f, Sh: %.0f" % (u.energy, u.shield),
```

Comment:
I guess there's currently no way to get the energy and shield percentages since I don't remember there being a shield_max and energy_max, though maybe it's in static_data? That might be nicer to represent than the absolutes. If it can be represented as a ratio, it may also be cool to do it as an arc around the unit, or a 2px bar instead of a number.